extra-network/v2ray-desktop/autobuild/prepare

```diff
@@ -1 +1,4 @@
 acbs_copy_git
+abinfo "Set the v2ray_usr_local_install..."
```

Comment:
This `abinfo` is not descriptive. It also does not address the second `sed` operation.